assets/css/custom.css

```
@@ -1,9 +1,100 @@
+}
+
+/* Two stacked icons per slot; `.is-locked` on the .finder-hero root, set by
+   script, picks the visible face. */
+.finder-swap-icon {
+  @apply relative inline-flex flex-none size-5;
+}
+
+.finder-swap-icon i,
+.finder-swap-icon img {
+  @apply absolute inset-0 flex items-center justify-center object-contain;
+  transition: opacity 0.4s ease;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)